astrokat/test/test_offline_observe.py

```diff
@@ -30,8 +30,8 @@ def test_targets_sim(self):
         result = LoggedTelescope.user_logger_stream.getvalue()
         self.assertIn("Single run through observation target list", result)
         self.assertIn("target0_radec observed for 10.0 sec", result)
-        self.assertIn("target1_azel observed for 10.0 sec", result)
-        self.assertIn("target2_gal observed for 10.0 sec", result)
+        self.assertIn("target1_gal observed for 10.0 sec", result)
+        self.assertIn("target2_azel observed for 10.0 sec", result)
 
```

Comment:
Shouldn't we include some verification for target3?
```
name=target3_azel, azel=10 50, tags=target, duration=10.0, type=scan
```

Reply:
Indeed, added a log message and validation test for scan target
```
self.assertIn("target3_azel, tags=radec target, 15:56:30.60 8:51:02.5", result)
```